server/conf/i18n/messages

```diff
@@ -97,6 +97,49 @@ title.submittedPrograms=Submitted
 toast.applicationSaved=Successfully saved application: application ID {0}
 toast.programCompleted=Application was already completed.
 
+#------------------------------------------------------------------------#
+# TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding a client #
+#------------------------------------------------------------------------#
+
+banner.clientInfoUpdated=Client info has been successfully updated.
+banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
+banner.acctDeleteError=This account cannot be deleted.
+
+button.addNewClient=Add new client
+button.backToEditing=Back to editing
+button.cancel=Cancel
+button.deleteAcct=Delete account
+button.keepAcct=Keep account
+button.nextPage=Next
+button.save=Save
+button.search=Search
+button.viewApplications=View applications
+
+content.numberOfAppSubmitted={0} applications submitted
+content.warning=There are no applications submitted for this client, so no applications will be lost.
+
+dialog.deleteConfirmation=Are you sure you want to delete this client's account?
+
+label.dob=Date of birth
+label.nameOrEmail=Name(s) or email
+label.notes=Notes
+label.contactInfo=Contact Information
+
+link.aboutCiviform=About CiviForm
```

Comment:
suggestion (non-blocking):  I don't think the page footer is part of the TI dashboard work.  It should be a separate effort to create the footer and internationalize it.  So, you probably don't need to include "About CiviForm", "Accessibility statement" and "Privacy Policy".